ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -1,44 +1,42 @@
-import React from 'react';
-import { useHistory, useParams } from 'react-router-dom';
-import { Button, PageSection } from '@patternfly/react-core';
+import React, { useCallback } from 'react';
+import { useParams } from 'react-router-dom';
+import { Divider, PageSection, Title } from '@patternfly/react-core';
 
-import { complianceEnhancedCoveragePath } from 'routePaths';
+import useRestQuery from 'hooks/useRestQuery';
+import { getComplianceProfileResults } from 'services/ComplianceResultsService';
 
+import PageTitle from 'Components/PageTitle';
 import CoveragesToggleGroup from './CoveragesToggleGroup';
 import CoveragesPageHeader from './CoveragesPageHeader';
+import ProfileChecksTable from './ProfileChecksTable';
 
 function ProfileChecksPage() {
-    const history = useHistory();
     const { profileName } = useParams();
 
+    const fetchProfileChecks = useCallback(
+        () => getComplianceProfileResults(profileName),
+        [profileName]
+    );
+    const { data: profileChecks, loading: isLoading, error } = useRestQuery(fetchProfileChecks);
+
     return (
         <>
+            <PageTitle title="Compliance coverage - Profile checks" />
             <CoveragesPageHeader />
             <PageSection>
                 <CoveragesToggleGroup tableView="checks" />
             </PageSection>
-            <PageSection variant="light">
-                <div>ProfileChecksPage</div>
-                <Button
-                    onClick={() => {
-                        history.push(
-                            `${complianceEnhancedCoveragePath}/profiles/${profileName}/clusters`
-                        );
-                    }}
-                    variant="primary"
-                >
-                    Go to all clusters (ProfileClustersPage)
-                </Button>
-                <Button
-                    onClick={() => {
-                        history.push(
-                            `${complianceEnhancedCoveragePath}/profiles/${profileName}/checks/test`
-                        );
-                    }}
-                    variant="primary"
-                >
-                    Go to single check (CheckDetailsPage)
-                </Button>
+            <PageSection variant="default">
+                <PageSection variant="light">
```

Comment:
Although not a blocker to merge, can we brainstorm a pattern for layout of pages?
* `<PageSection>…</PageSection><PageSection><PageSection>…` looks complicated.
* What do you think about `component="div"` prop when `PageSection` is for layout only, so `section` element in HTML is semantic?

Reply:
I'm more than open to discussing alternatives. I didn't like doing this either but it felt like the best option that didn't involve adding css at the container levels. I either forgot or didn't know about the component prop for `PageSection` – that works and looks good.